### PR TITLE
Add new redhat-no-hermetic ECP

### PR DESCRIPTION
**Pull request by lcarva**
https://issues.redhat.com/browse/HACBS-2657

### CODE REVIEW

**Review by simonbaird — APPROVED** [review]

**Review by zregvart — APPROVED** [review]

File: components/enterprise-contract/ecp.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ spec:
   sources:
     - data:
         - github.com/release-engineering/rhtap-ec-policy//data?ref=be7e1ef73bdeef2752dde400a52f9eab9b7542ca
-        - oci::quay.io/redhat-appstudio-tekton-catalog/data-acceptable-bundles:latest@sha256:85431601d5d50520d6de63f649d37943a68c86de0a62228dab30a7774fa074c0
+        - oci::quay.io/redhat-appstudio-tekton-catalog/data-acceptable-bundles:latest@sha256:dc89022688fef704e1d092d312ebceaf457e4b07594746c3dfb1231eb71e452a
```
[the remark this answers is not in the document]
Comment (lcarva) [reply components/enterprise-contract/ecp.yaml:27]: Yeah. Added a message. Thanks!

**Pull request merged**